pkg/gofr/datasource/file/s3/interface.go

```diff
@@ -12,6 +14,15 @@ type Logger interface {
 	Errorf(pattern string, args ...interface{})
 }
 
+type s3conn interface {
```

Comment:
Can you rename it? Thanks


```suggestion
type s3Conn interface {
```